python/tchannel/scheme.py

```diff
@@ -41,10 +62,18 @@ def __init__(self):
     def type(self):
         return self._type
 
-    def deserialize(self, obj):
+    def deserialize_header(self, obj):
+        assert obj
+        return json.loads(obj)
+
+    def serialize_header(self, obj):
+        assert obj
+        return json.dumps(obj)
```

Comment:
It's totally possible to serialize None. No reason to assert here.  `json.dumps` knows how to handle `None`.